### ISSUE TITLE
Option to include scss body as type `Styles.myClass: string` would become `Styles.myClass: 'color: white;'`

### ISSUE THREAD

**Issue by flowreaction** [skovy/typed-scss-modules#194]
I was wondering if this package is still being developed with new features? I am currently implementing this package in a codebase, and a nice addition to the current functionality could be a string literal type for each class name containing the actual SCSS code. So 

```scss
.myClass {
  color: white;
}
```

could be: 

```ts
import Styles from 'myStyles.module.scss'

Styles.myClass
//         ^ this here would be of type: `color: white;` instead of `string`
```
This way, you could hover over the properties and see which scss is actually behind the class name.

**Comment by flowreaction** [comment]
I would also be willing to write a PR for this option if this would be a nice enhancement 😄

**Comment by flowreaction** [comment]
POC PR here #195

**Comment by skovy** [comment]
I can see the benefit of having the styles surfaced directly in the editor, but I'm not sure this would be a widely used feature since it could have a number of downsides.

- Any style change will break the generated types since the generated types would not effectively include the entirety of the styles.
- The types are technically incorrect, since the type of `myClass` is a string that is that class (something like `myClass_[HASH]` and not the raw styles).

I'm also not sure if this tool is the right place to solve this problem. This tool is aimed at generating type definitions for SCSS Modules. This feature would be aimed more at improving the developer experience of working with CSS Modules. That makes me wonder if this is more of a VSCode plugin that provides this functionality?

If it were in this package, another option could be inline this as a JSDoc comment.

I'm not opposed to this functionality in this package, but I'm not fully convinced it should live here.

**Comment by flowreaction** [comment]
Fair enough! Might be something that is better suited for a CSS modules VS Code extension or something along the lines, how the tailwind VS Code extension handles this.

Anyway, thanks for this package!
